ThunderCatsTests/UIAlertControllerTests.m

```diff
@@ -0,0 +1,66 @@
+//
+//  UIAlertControllerTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-09.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UIAlertController+TCAdditions.h"
+
+@interface UIAlertControllerTests : XCTestCase
+
+@end
+
+@implementation UIAlertControllerTests
+
+- (void)testNotImplementedAlert
+{
+    UIAlertController *alert = [UIAlertController tc_notImplementedAlert];
+    
+    XCTAssertEqualObjects(alert.title, @"Not Implemented");
+    XCTAssertEqualObjects(alert.message, @"This feature is not yet implemented.");
+    XCTAssertEqualObjects(alert.actions[0].title, @"Ok");
+}
+
+- (void)testAlertWithTitle
```

Comment:
I tried adding a completion action. I just assigned a value to a string just so it had something to do. The title etc still pass but the string I assigned a value to comes out as nil so I think I am doing the cancel action incorrectly still (we talked about this in the helpline a few days ago):

`    __block NSString *cancelActionString = @"";
    UIAlertController *alertWithCancelAction = [UIAlertController tc_alertWithTitle:title message:message cancelButtonTitle:cancelButtonTitle cancelAction:^(UIAlertAction * __tc_nonnull cancelAction) {
        cancelActionString = @"Successful";
    }];
    
    XCTAssertEqual(alertWithCancelAction.title, title);
    XCTAssertEqual(alertWithCancelAction.message, message);
    XCTAssertEqual(alertWithCancelAction.actions[0].title, cancelButtonTitle);
    XCTAssertEqualObjects(cancelActionString, @"Successful");`